tests/DocumentationExamplesTest.php

```diff
@@ -925,6 +925,41 @@ public function testExample_55_58()
         $this->assertInventoryCount(0);
     }
 
+    public function testExample_59_61()
+    {
+        $db = new Database($this->manager, $this->getDatabaseName());
+
+        // Start Changestream Example 1
+        $cursor = $db->inventory->watch();
+        $cursor->next();
+        $current = $cursor->current();
+        // End Changestream Example 1
+
+        $this->assertNull($current);
+
+        // Start Changestream Example 2
+        $cursor = $db->inventory->watch([], ['fullDocument' => \MongoDB\Operation\ChangeStreamCommand::FULL_DOCUMENT_UPDATE_LOOKUP]);
+        $cursor->next();
+        $current = $cursor->current();
+        // End Changestream Example 2
+
+        $this->assertNull($current);
+        $db->inventory->insertOne(['x' => '1']);
+        $cursor->next();
+        $current = $cursor->current();
+
+        // Start Changestream Example 3
+        $resumeToken = ($current !== null) ? $current->_id : null;
+        if ($resumeToken !== null) {
+            $cursor = $db->inventory->watch([], ['resumeAfter' => $resumeToken]);
+            $cursor->next();
+        }
+
+        // End Changestream Example 3
+
+        $this->assertNull($cursor->current());
```

Comment:
One more thought. Let's `$db->inventory->insertOne(['x' => 2]);` here, so we can assert a non-null change entry. I'm not sure if you're doing so in #428 yet, but if you know the structure of the change entry you can use a more specific assertion. We should have helpers in the base test classes that can be used to assert an equivalent document structure.